charts/hivemq-operator/values.yaml

```diff
@@ -187,11 +187,7 @@ hivemq:
       enabled: false
       # Note that this is just an example initialization routine. Make sure this points to the current JDBC version you require for your configuration.
       initialization: |
-        # A little hack because k8s configMaps can't handle sub-directories
```

Comment:
All released HiveMQ versions still support the old config location, and the older versions (4.9 etc.) are _only_ supporting the old config location.

So I think this is the correct way to handle this.

Reply:
Please add the answer to this comment here, for future reference

Reply:
Yes this was added to avoid the warnings, however it increases complexity to understand which config is being used.